### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by yalh76**
## Problem

- *Level0*

## Solution

- *Apply last example_ynh*
- *Few fixes and improvments*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

**Comment by nicofrand** [comment]
Hi!

Thanks for doing this but we are losing some precious information here in the Readme (regarding the demo at least), and what is missing etc.

**Comment by yalh76** [comment]
I've updated the disclaimer with previous informations, you can edit it if needed

File: README.md [hunk 1]
```diff
@@ -56,14 +53,12 @@ Kresus is an open-source libre self-hosted personal finance manager. It allows y
...
- * Report a bug about this package: https://github.com/YunoHost-Apps/kresus_ynh
- * Report a bug about Kresus itself: https://framagit.org/kresusapp/kresus
- * Kresus website: https://kresus.org
- * YunoHost website: https://yunohost.org
-
----
+* Official app website: https://framagit.org/kresusapp/kresus
+* Upstream app code repository: https://framagit.org/kresusapp/kresus
+* YunoHost documentation for this app: https://yunohost.org/app_kresus
+* Report a bug: https://github.com/YunoHost-Apps/kresus_ynh/issues
```
Comment (nicofrand) [inline README.md:61]: Can we add back `* Official documentation: https://kresus.org/en/doc.html` please?

**Comment by nicofrand** [comment]
Thanks for the update! One small comment re. the official doc and I think that should be great

File: README.md (hunk 1 above)
Comment (yalh76) [reply README.md:62]: Added ;)

**Pull request merged**